.circleci/config.yml

```diff
@@ -76,3 +120,6 @@ workflows:
       - staging-test:
           requires:
             - docs-lint
+      - staging-test-with-rebase:
```

Comment:
I agree that this is a good practice (running tests both on their own branch and rebasing on the target branch - this is what we do for Travis tests as well).

Reply:
Though this increases costs so @msheiny/@conorsch happy to re-evaluate later this week